frontend/src/components/react/pages/ChainMemberList.tsx

```diff
@@ -1551,6 +1551,20 @@ function RouteTable(props: {
   const [dragging, setDragging] = useState<string>("");
   const [dragTarget, setDragTarget] = useState<string>("");
 
+  function mapUsersToOrder(users: { uid: string }[]): { [uid: string]: number } {
+    return users.reduce((acc, user, index) => {
+      acc[user.uid] = index + 1;
+      return acc;
+    }, {} as { [uid: string]: number });
+  }
+
+  const [routeOrder,setRouteOrder] = useState<{[uid: string]: number}>(() => mapUsersToOrder(props.users));
```

Comment:
Please change this to a useMemo that way you don't need the useEffect lower down